docs/source/publish_and_share_kedro_viz_on_gcp.md

```diff
@@ -0,0 +1,115 @@
+# Publish and share via GCP
+
+This page describes how to publish Kedro-Viz to Azure to share it with others. It uses the spaceflights tutorial as an example.
```

Comment:
small typo here - instead of Azure it should say GCP for the 'Publish and share via GCP' page

Reply:
how to publish Kedro-Viz **on** GCP